app/assets/javascripts/app/utils/auto-logout.js

```diff
@@ -1,5 +1,3 @@
 export default () => {
-  window.onbeforeunload = null;
```

Comment:
I think it's important to have a null `window.onbeforeunload`

One of the reasons for doing this was that the "are you sure you want to leave this page" was getting in the way of some of our session timeout stuff: https://github.com/18F/identity-private/issues/1201#issuecomment-267979900

I had some PRs to clean this up:
- https://github.com/18F/identity-idp/pull/990
- https://github.com/18F/identity-idp/pull/991


I guess I just want to make sure this doesn't cause any regressions